network-api/networkapi/wagtailpages/factory/youtube_regrets_page.py

```diff
@@ -30,6 +34,25 @@ class Meta:
     regret_stories = Faker("streamfield", fields=["regret_story"] * 28)
 
 
+class AppInstallPageFactory(PageFactory):
```

Comment:
This factory looks and works great! 👍

Since this page isn't specific to YouTube Regrets (e.g., we also use it for a page related to TikTok on another part of our site), could we move this factory to its own file, named something like `app_install_page.py`? This would allow us to import the factory here to create the YouTube Regrets Reporter extension page as you've done on lines [135-159](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12581/files#diff-7a3dd1d67d821c302b9ae7d85764ae697d17909ca513cf6264188ebdaa3ed58bR135-R159), and it would also make it easier to import it wherever else we might need it in the future.